source/FreeRTOS_DHCPv6.c

```diff
@@ -396,7 +332,7 @@ static BaseType_t xDHCPv6Process_PassReplyToEndPoint( struct xNetworkEndPoint *
         pxIterator = pxIterator->pxNext;
     }
 
-    if( ( pxIterator != NULL ) && ( pxIterator->pxDHCPMessage != NULL ) )

```

Comment:
pxDHCPMessage is always allocated at the initial state. And trigger assertion if allocate fails.
So we don't need to check it here.